controllers/space/space_controller_test.go

```diff
@@ -901,9 +901,12 @@ func TestUpdateSpaceTier(t *testing.T) {
 	err := apis.AddToScheme(s)
 	require.NoError(t, err)
 	base1nsTier := tiertest.Base1nsTier(t, tiertest.CurrentBase1nsTemplates)
+	base1nsTier.Status.Revisions = tiertest.Base1nsRevision
 	// get an older base1ns tier (with outdated references)
 	olderbase1nsTier := tiertest.Base1nsTier(t, tiertest.PreviousBase1nsTemplates)
+	olderbase1nsTier.Status.Revisions = tiertest.OldTierRevision
 	otherTier := tiertest.OtherTier()
+	otherTier.Status.Revisions = tiertest.OtherTierRevision
```

Comment:
why setting the revisions separately, and not as part of the functions that create the tiers?